lldb/source/Plugins/SymbolFile/NativePDB/SymbolFileNativePDB.cpp

```diff
@@ -1810,7 +1813,27 @@ SymbolFileNativePDB::ParseVariablesForCompileUnit(CompileUnit &comp_unit,
                                                   VariableList &variables) {
   PdbSymUid sym_uid(comp_unit.GetID());
   lldbassert(sym_uid.kind() == PdbSymUidKind::Compiland);
-  return 0;
+  for (const uint32_t gid : m_index->globals().getGlobalsTable()) {
+    PdbGlobalSymId global{gid, false};
+    CVSymbol sym = m_index->ReadSymbolRecord(global);
+    // TODO: Handle S_CONSTANT which might be a record type (e.g.
+    // std::strong_ordering::equal). Currently
+    // lldb_private::npdb::MakeConstantLocationExpression doesn't handle this
+    // case and will crash if we do create global variables from it.
```

Comment:
Is there some simple fix which would make it not crash at least?

Reply:
Just what I did: not handling the S_CONSTANT case prevents the call to `lldb_private::npdb::MakeConstantLocationExpression` with a record type, hence not crash for now.

Reply:
Like Pavel I understood the comment as if we'd still crash. Maybe update the wording to make it explicit that that's the reason we don't handle it here.

Reply:
Updated wording.